src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +53,26 @@
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
```

Comment:
Ensure that you pass the correct class type to the injector to avoid `ClassCastException`. Double-check that `MovieSessionService.class` is the correct type for this instance.